helios-services/src/main/java/com/spotify/helios/master/MasterParser.java

```diff
@@ -79,9 +107,46 @@ protected void addArgs(final ArgumentParser parser) {
         .type(Integer.class)
         .setDefault(5802)
         .help("admin http port");
+
+    authenticationScheme = parser.addArgument("--auth-scheme")
+        .metavar("SCHEMENAME")
+        .help("Name of authentication scheme to use. "
+              + "Setting this flag enables authentication in Helios. "
+              + "'crtauth' support is built into Helios. "
```

Comment:
Should we also mention `http-basic` here or is the implementation too "basic" to be of any real use?